dts/bindings/serial/st,stm32-uart-base.yaml

```
@@ -4,7 +4,12 @@
 # Common fields for STM32 UART peripherals.
 description: STM32 UART-BASE
 
-include: [uart-controller.yaml, pinctrl-device.yaml, reset-device.yaml]
+include:
+  - name: uart-controller.yaml
+    property-blocklist:
+      - clock-frequency
+  - pinctrl-device.yaml
+  - reset-device.yaml
```

Naive question, shouldn't is be
```suggestion
  - name: pinctrl-device.yaml
  - name: reset-device.yaml
```
?

---

The yaml doesn't require that, but it can be done for consistency. Shall I?

---

For consistency it would be nice indeed (and avoids questions on the next one reading these lines)